core/src/models/Schedule.ts

```diff
@@ -356,6 +355,16 @@ export class Schedule extends CommonModel<Schedule> {
     }
   }
 
+  @BeforeValidate
+  static async ensureSetIncremental(instance: Schedule) {
+    if (instance.isNewRecord && typeof instance.incremental === "undefined") {
```

Comment:
Could it ever be null?  I guess it depends on how the not-yet-saved instance was created

Reply:
Yeah, if `null` was set on an action or config, sequelize will catch it and say it can't be null. I wanted to keep this value-setting specifically to when it hasn't been provided.